src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/OnOffClientFragment.kt

```diff
@@ -59,6 +63,7 @@ class OnOffClientFragment : Fragment() {
 
   override fun onStart() {
     super.onStart()
+    fabricIdEd.setText(5544332211.toString())
```

Comment:
This is odd ... can we just set it as a string?

Reply:
Another question is why is this baked-in all the way down here?

Reply:
It's temporary. Eventually, once we have a method for generating fabric ID that is to be used for commissioning all devices, we will use it to initialize this field. But AFAIK there's no such thing for now.

Reply:
should this have a TODO comment above?